ng-gridify.component.ts

```diff
@@ -16,7 +16,7 @@ import { ngGridifyData } from './ng-gridify.types';
     </tr>
     <tr *ngFor="let row of gridData.Data | PagePipe: itemsPerPage : currentPage : sortByColumn : sortByAscending">
       <td *ngFor="let col of columnKeys;">
-        {{row[col]}}
+        {{DeepValue(row, col)}}
```

Comment:
This is great, but at the moment if I use the dot notation to get a deep property, if that property does not exist, I get a "Cannot read property '[Property Name]' of undefined" error. 

I'm happy to merge this if there is sufficient error checking in place to suppress any errors that might occur.

As an example, if I use the following configuration object I currently get an error.

{        
      Title: 'The Grid Title', 
      ExportEnabled: true,
      Columns: [
        { Name: 'Id', DisplayValue: 'Id' },
        { Name: '**DeepProperty.Name**', DisplayValue: 'Name' },        
        { Name: 'Job', DisplayValue: 'Job' }
      ],
      Data: [
        { Name: 'Frank Reynolds', Id: 1, Job: 'Mastermind', **DeepProperty : { Name: 'IAIN!'}** },
        { Name: 'Deandra Reynolds', Id: 2, Job: 'Bartender' }    
      ], 
      ItemClick: {
        Function: function (item) { 
            alert( item.Name + ', ' + item.Job);
        },
        Text: 'Click me to see a summary'
      },
      ItemsPerPage: 5
    }